src/mlpack/methods/xgboost/loss_functions/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Define the files we need to compile.
+# Anything not in this list will not be compiled into mlpack.
+set(SOURCES
+  sse_loss.hpp
```

Comment:
I think you could just use `loss_functions/sse_loss.hpp` in the parent directory's `CMakeLists.txt`, and then you don't need this file. :+1:

Reply:
Great! I will apply this change.